clients/admin/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "admin"
-version = "0.5.6"
```

Comment:
there's a handful of these unintentional downgrades of code, once over this whole PR